2/contrib/s2i/run

```diff
@@ -151,12 +151,14 @@ CONTAINER_MEMORY_IN_MB=$((CONTAINER_MEMORY_IN_BYTES/2**20))
 if [[ "${OPENSHIFT_JENKINS_JVM_ARCH}" == "x86_64" || \
     ( "${OPENSHIFT_JENKINS_JVM_ARCH}" == "" && "${CONTAINER_MEMORY_IN_MB}" -ge 2048 ) ]]; then
   alternatives --set java $(alternatives --display java | awk '/family.*x86_64/ { print $1; }')
+  alternatives --set javac $(alternatives --display javac | awk '/family.*x86_64/ { print $1; }')
 else
   alternatives --set java $(alternatives --display java | awk '/family.*i386/ { print $1; }')
+  alternatives --set javac $(alternatives --display javac | awk '/family.*i386/ { print $1; }')
   export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
 fi
 
-echo "OPENSHIFT_JENKINS_JVM_ARCH='${OPENSHIFT_JENKINS_JVM_ARCH}', CONTAINER_MEMORY_IN_MB='${CONTAINER_MEMORY_IN_MB}', using $(readlink /etc/alternatives/java)"
+echo "OPENSHIFT_JENKINS_JVM_ARCH='${OPENSHIFT_JENKINS_JVM_ARCH}', CONTAINER_MEMORY_IN_MB='${CONTAINER_MEMORY_IN_MB}', using $(readlink /etc/alternatives/java) and $(readlink /etc/alternatives/java)"
```

Comment:
did you intend for this to print two different values? both are referencing java right now, not javac.

Reply:
yeah the second should be javac ... will update in a moment